tests/test_text_generation_example.py

```diff
@@ -31,6 +31,7 @@
             ("meta-llama/Llama-2-7b-hf", 512, False, 8711),  # in some cases like TGI, reuse_cache isnt used
             ("stabilityai/stablelm-2-12b", 1, False, 81.842),
             ("codellama/CodeLlama-34b-hf", 1, True, 32.644),
+            ("Qwen/Qwen1.5-7B", 1, False, 451.7454544774087),
```

Comment:
Isn't this result for batch size 4?

Reply:
Oh, I see, BTW, what True\False stands for?

Reply:
https://github.com/huggingface/optimum-habana/blob/main/tests/test_text_generation_example.py#L169
It looks like weather to use reuse_cache or not.